tests/tools/test_utils.py

```diff
@@ -51,3 +56,24 @@ class FakeLoader:
 
     with patch("dissect.target.tools.utils.LOADERS_BY_SCHEME", {"loader": FakeLoader}):
         assert args_to_uri(targets, loader_name, rest) == uris
+
+
+@pytest.mark.parametrize(
+    "pattern, expected_function",
+    [
+        ("passwords", "dissect.target.plugins.os.unix.shadow.ShadowPlugin"),
+        ("firefox.passwords", "Unsupported function `firefox` for target"),
+    ],
+)
+def test_plugin_name_confusion_regression(target_unix_users, pattern, expected_function):
```

Comment:
This was an issue? I don't immediately see something in this PR that fixes this, or is this a precautionary regression test?